napari/layers/surface/surface.py

```diff
@@ -355,6 +360,42 @@ def shading(self, shading):
             self._shading = Shading(shading)
         self.events.shading(value=self._shading)
 
+    @property
+    def wireframe(self):
```

Comment:
```suggestion
    def wireframe(self) -> SurfaceWireframe:
```
for better IDE support